docs/build/javascript/core.md

```diff
@@ -33,19 +33,16 @@ import Ceramic from '@ceramicnetwork/core'
 Ceramic utilizes the [dag-jose](../../learn/glossary.md#dagjose) IPLD codec to format and store data in IPFS.
 
 ```javascript
-import Ipfs from 'ipfs-core'
+import * as Ctl from 'ipfsd-ctl'
 import dagJose from 'dag-jose'
-import { convert } from 'blockcodec-to-ipld-format'
-
-const dagJoseFormat = convert(dagJose)
 ```
 
 ## **4. Create an IPFS instance**
 
-Create an instance of `js-ipfs` with `dag-jose` enabled.
+Create an instance of `go-ipfs` with `dag-jose` enabled.
 
 ```javascript
-const ipfs = await Ipfs.create({ ipld: { formats: [dagJoseFormat] } })
+const ipfs = await Ctl.createController({ ipfsOptions: { ipld: { codecs: [dagJose] } } }
```

Comment:
Actually, `dag-jose` is enabled by default in `go-ipfs` and does not need to be enabled explicitly. I'll fix this,